content/authoring/kata/create-first-kata.md

```diff
@@ -53,23 +53,23 @@ This is bad because creating a kata about them again constitutes a **duplicate**
 
 As we all know, before you even engage your enemies, you need to send scouts forward.
 
-For example, while [Peano](https://www.codewars.com/kata/search/?q=peano) and [Church](https://www.codewars.com/kata/search/?q=church) numbers are definitely not easy, as the search results clearly shows, they've been done many times already.
+For example, while [Peano](https://www.codewars.com/kata/search/?q=peano) and [Church](https://www.codewars.com/kata/search/?q=church) numbers are definitely not easy, as the search results clearly show, they've been done many times already.
 
-In a way, knowing the kata in the wild is very similar to knowing your enemies: you get to see how others write the same kata, other people's (sometimes) brilliant solution in these kata.
+In a way, knowing the kata in the wild is very similar to knowing your enemies: you get to see how others write the same kata and other people's (sometimes) brilliant solution in these kata.
 
-Then, maybe you can decide whether to retreat, or make a **harder** version of the kata as return if you feel the existing one is too lackluster/easy.
+Then, maybe you can decide whether to retreat or make a **harder** version of the kata as a fallback if you feel the existing one is too lackluster/easy.
```

Comment:
I'm not sure I understand the idea of creating a harder version of a Kata as "a fallback" ...

Maybe change the wording from "as return" to "in return" instead?

Reply:
```suggestion
Then, maybe you can decide whether to retreat or make a **harder** version of the kata in return if you feel the existing one is too lackluster/easy.
```